photon-ml/src/main/scala/com/linkedin/photon/ml/data/KeyValueScore.scala

```diff
@@ -63,7 +63,12 @@ protected[ml] class KeyValueScore(val scores: RDD[(Long, Double)]) extends RDDLi
     val addedScores =
```

Comment:
Can we factor out the code for the addition and subtraction, passing in a lambda for the operation to perform? Only the +/- changes between these 2 functions.